lib/models/timetable/course_model.dart

```diff
@@ -9,17 +9,24 @@ class CourseModel implements Comparable<CourseModel> {
   String? venue;
   String? midsem;
   String? endsem;
+  String? midSemVenue;
+  String? endSemVenue;
+  Map<String, dynamic>? timings;
```

Comment:
why are there 2 fields timings and timing

Reply:
removed timing field